portal/docs/overview/report_payments.md

```diff
@@ -3,27 +3,26 @@ sidebar_position: 4
 ---
 
 # Using the API to report repayments
-
-We recommend that your technical team consults with the Kiva Coordinator at your organization to fully understand Kiva's Repayment Reporting process. To do so, please have your technical team review the following:
-
-
-This video provides a thorough explanation of the repayment reporting process.
```

Comment:
The video link was done differently here than on ZenDesk: instead of providing a hyperlink, there is a thumbnail link. That's fine and maybe better, as far as I'm concerned, but I did collapse this line into the previous paragraph to cut down on verbage.

I can't make a similar editorial call on the FR/ES pages, but they use hyperlinks. I think this small discrepancy between the pages is fine for now; the content is effectively the same.